include/boost/geometry/strategies/cartesian/box_in_box.hpp

```diff
@@ -61,9 +65,94 @@ struct box_covered_by_range
 };
 
 
+// spherical_equatorial_tag and spherical_polar_tag is casted to spherical_tag
+template <typename Geometry>
+struct box_within_range<Geometry, 0, spherical_tag>
+{
+    template <typename BoxContainedValue, typename BoxContainingValue>
+    static inline bool apply(BoxContainedValue const& bed_min,
+                             BoxContainedValue const& bed_max,
+                             BoxContainingValue const& bing_min,
+                             BoxContainingValue const& bing_max)
+    {
+        typedef typename select_most_precise
+            <
+                BoxContainedValue,
+                BoxContainingValue
+            >::type calc_t;
+        typedef typename coordinate_system<Geometry>::type::units units_t;
+        typedef math::detail::constants_on_spheroid<calc_t, units_t> constants;
+
+        // min <= max <=> diff >= 0
+        calc_t const diff_ed = bed_max - bed_min;
+        calc_t const diff_ing = bing_max - bing_min;
+        // if containing is smaller it cannot contain
+        // or interiors doesn't overlap (no interior in contained)
+        if (diff_ing < diff_ed || diff_ed == 0)
+            return false;
+
+        // if containing covers the whole globe it contains all
+        if (!math::smaller(diff_ing, constants::period())) // >= period
+            return true;
+
+        // calculate positive longitude translation with bing_min as origin
+        calc_t const c0 = 0;
+        calc_t diff_min = bed_min - bing_min;
+        math::normalize_longitude<units_t, calc_t>(diff_min);
+        if (diff_min < c0) // [-180, 180] -> [0, 360]
+            diff_min += constants::period();
+
+        return bing_min + diff_min + diff_ed <= bing_max;
+    }
+};
+
+
+template <typename Geometry>
+struct box_covered_by_range<Geometry, 0, spherical_tag>
+{
+    template <typename BoxContainedValue, typename BoxContainingValue>
+    static inline bool apply(BoxContainedValue const& bed_min,
+                             BoxContainedValue const& bed_max,
+                             BoxContainingValue const& bing_min,
+                             BoxContainingValue const& bing_max)
```

Comment:
Sorry, I have no idea what you mean by bed / bing. Can you change the names to something more meaningful?